.github/workflows/ci-x.yml

```
@@ -36,23 +34,26 @@ jobs:
       MAVEN_OPTS: -Xmx2048m
     steps:
       - uses: actions/checkout@v2
+
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            !~/.m2/repository/org/keycloak
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Download Maven Repo
-        uses: actions/download-artifact@v1
-        with:
-          name: maven-repo
-          path: .
-      - name: Extract Maven Repo
-        shell: bash
-        run: tar -xzvf maven-repo.tgz -C ~
+
       - name: Run Quarkus unit tests
-        run: mvn clean install -B -Pquarkus -f quarkus/pom.xml
-      - name: Build testsuite
-        run: mvn clean install -B -Pquarkus,auth-server-quarkus -DskipTests -f testsuite/pom.xml
-      - name: Run base tests
-        run: mvn clean install -B -Pquarkus,auth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml -Dtest='!org.keycloak.testsuite.adapter.**,!**.crossdc.**,!**.cluster.**' | misc/log/trimmer.sh; exit ${PIPESTATUS[0]}
+        run: |
+          mvn clean install -B -Pquarkus -f quarkus/pom.xml
+          mvn clean install -B -Pquarkus,auth-server-quarkus -DskipTests -f testsuite/pom.xml
+          mvn clean install -B -Pquarkus,auth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml -Dtest='!org.keycloak.testsuite.adapter.**,!**.crossdc.**,!**.cluster.**' | misc/log/trimmer.sh; exit ${PIPESTATUS[0]}
```

Just cosmetic change to be consistent, somewhere the `exit ${PIPESTATUS[0]}` is on separate line and somewhere not
```suggestion
          mvn clean install -B -Pquarkus,auth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml -Dtest='!org.keycloak.testsuite.adapter.**,!**.crossdc.**,!**.cluster.**' | misc/log/trimmer.sh
          exit ${PIPESTATUS[0]}
```